pd/src/state.rs

```diff
@@ -289,14 +314,15 @@ INSERT INTO blobs (id, data) VALUES ('gc', $1)
 
         let mut validators: BTreeMap<tendermint::PublicKey, Validator> = BTreeMap::new();
 
-        let stored_validators = query!(r#"SELECT tm_pubkey, voting_power FROM validators"#)
+        let stored_validators = query!(r#"select tm_pubkey, validator_rates.voting_power FROM validators LEFT JOIN validator_rates ON validator_rates.validator_pubkey = validators.tm_pubkey;"#)
```

Comment:
non-blocking: later, we should set a validator limit in the genesis app state, and use it to limit the query here (selecting the top N validators by voting power).